dagql/dagui/opts.go

```diff
@@ -102,3 +116,45 @@ func (opts FrontendOpts) ShouldShow(span *Span) bool {
 	}
 	return true
 }
+
+func ShouldSkipFunction(obj, field string) bool {
+	// TODO: make this configurable in the API but may not be easy to
```

Comment:
I have some code in https://github.com/dagger/dagger/pull/9327 for filtering that would be helpful here:

https://github.com/dagger/dagger/pull/9327/files#diff-103e59fe01bf9664b34164f42a0052ad7c2432a4e3472900e99fe611b915b3d9

Feel free to carry on with this though - I can re-integrate + de-couple later.

I should try to land that PR though - maybe I'll just mark the API experimental for now.

Reply:
Ahh, that definitely looks useful :eyes: Being able to filter it better would definitely help.

Any objection to landing this in the meantime though, I'm happy to refactor it to match the newer API once it lands :tada: